hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/GenesisRecordsConsensusHook.java

```diff
@@ -68,7 +72,10 @@ public void process(@NonNull final TokenContext context) {
         final var blockStore = context.readableStore(ReadableBlockRecordStore.class);
 
         // This process should only run ONCE, when a node receives its first transaction after startup
-        if (blockStore.getLastBlockInfo().consTimeOfLastHandledTxn() != null) return;
+        //        if (blockStore.getLastBlockInfo().consTimeOfLastHandledTxn() != null) return;
```

Comment:
This absolutely has to be part of the code. The only time we can publish synthetic records is on the first consensus timestamp we get after starting up the node. There should be no need to publish more synthetic records later.

Reply:
@mhess-swl 
- we never get past the check for `consTimeOfLastHandledTxn() != null`
- a condition that I added works as expected -- synthetic records being published only once after starting the node (but I think there should be a better way)